### DISCUSSION TITLE
ZIO integration with Ethereum

### DISCUSSION THREAD

**Discussion by artur-jablonski** [zio/zio#7127] [Q&A]
I am exploring talking to Ethereum node JSON-RPC API using zio-http as transport layer. 

I found this project for scala code generation from ABIs (https://github.com/Lbqds/eth-abi) that mentions ZIO but it looks dead. 

Anybody knows of any ongoing effort for "Ethereum client backed by ZIO" kind of thing?

**Comment by adamgfraser** [comment]
@artur-jablonski I haven't but that would be really cool! Let me know if there is anything I can do to help.
